masterlist.yaml

```
@@ -2352,6 +2360,9 @@ plugins:
     url: [ 'https://www.nexusmods.com/fallout4/mods/24461' ]
     req:
       - 'Radrose Usability Enhancements.esp'
+  # AAF 
+  - name: 'AAF.esm'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/31304' ]
```

Generally an entry isn't added for location only.
It does have a requirement F4SE that could be added.
```suggestion
  # AAF 
  - name: 'AAF.esm'
    url: [ 'https://www.nexusmods.com/fallout4/mods/31304' ]
    req: [ *F4SE ]
```